docs/docsite/rst/rules/default_rules.rst

```diff
@@ -56,4 +56,10 @@ ID
 `E704 <https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/rules/MetaVideoLinksRule.py>`_                                                                                                                                                                                                    v4.0.0                                                                                                                                                                                                                                                                                                       meta/main.yml video_links should be formatted correctly                                                                                                                                                                                                                                                      Items in ``video_links`` in meta/main.yml should be dictionaries, and contain only keys ``url`` and ``title``, and have a shared link from a supported provider
+
+**E9xx - core**
+`E901 <https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/rules/LoadingFailureRule.py>`_                                                                                                                                                                                                    v4.3.0                                                                                                                                                                                                                                                                                                       Failed to load or parse file                                                                                                                                                                                                                                                                                 Linter failed to process a YAML file, possible not an Ansible file.
+
+**E9xx - core**
```

Comment:
This is caused by the fact that the class is loaded twice, is a bug docs generate logic, wait for https://github.com/ansible/ansible-lint/pull/779 to merge and we should not see it anymore.